### PR TITLE
KINNO-643-open_module_activity

### PR DESCRIPTION
**Pull request by shayb**
#### Main purpose:
add transfer item on click show module activity
#### Technical description:
start activity
#### Dilemmas you faced with? 
item only visible to kin 3 users 
#### Tests:
(Main tests you added/changed)
#### Documentation:
(Source/readme.md)

### CODE REVIEW

File: sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt [hunk 1]
```diff
@@ -83,6 +82,15 @@ class SettingsFragment : KinEcosystemBaseFragment<ISettingsPresenter, ISettingsV
...
     }
 
+    override fun showTransferItem(show: Boolean) {
+        if(show) {
+            transferItem.visibility = VISIBLE
+        }
+        else {
+            transferItem.visibility = GONE
+        }
+    }
```
Comment (OrenZak) [inline sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt:87]: can be simplified to one line:
`transferItem.visibility = if (show) VISIBLE else GONE`

File: sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt [hunk 2]
```diff
@@ -119,6 +127,17 @@ class SettingsFragment : KinEcosystemBaseFragment<ISettingsPresenter, ISettingsV
         presenter?.onActivityResult(requestCode, resultCode, data)
     }
 
+    override fun startTransferFlow() {
+        context?.let {
+            try {
+                startActivity(AppsTransferActivity.getIntent(it))
+                activity?.overridePendingTransition(R.anim.kinecosystem_slide_in_right, R.anim.kinecosystem_slide_out_left)
+            } catch (e: Exception) {
+                //activity not found
```
Comment (OrenZak) [inline sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt]: should we show some Toast?

**Review by OrenZak — APPROVED** [review]
See my comments.

File: sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt (hunk 2 above)
Comment (shayb) [reply sdk/src/main/java/com/kin/ecosystem/settings/view/SettingsFragment.kt]: yes toast could be nice

**Pull request merged**